kafka-connect/kafka-connect/src/main/java/org/apache/iceberg/connect/IcebergSinkConfig.java

```diff
@@ -79,6 +79,8 @@ public class IcebergSinkConfig extends AbstractConfig {
       "iceberg.tables.schema-force-optional";
   private static final String TABLES_SCHEMA_CASE_INSENSITIVE_PROP =
       "iceberg.tables.schema-case-insensitive";
+  private static final String TABLES_SCHEMA_FORCE_COLUMNS_TO_LOWERCASE_PROP =
+      "iceberg.tables.schema-force-columns-to-lowercase";
```

Comment:
I think `iceberg.tables.schema-force-lowercase` is sufficient and consistent with `schema-force-optional`. Also, moving this up one to be below `schema-force optional` would be preferable.